lib/avo/resources/base.rb

```diff
@@ -307,6 +319,13 @@ def detect_fields
         }
       end
 
+      unless defined? VIEW_CARDS_MAPPING
+        VIEW_CARDS_MAPPING = {
+          index: [:index_cards, :display_cards],
+          show: [:show_cards, :display_cards]
+        }
+      end
```

Comment:
This is going in the right direction, but it's currently incomplete. The form views are still missing. You probably assumed that cards are meant to be displayed only on the show view, but that's not the case. Avo provides the ability to display cards on all views. Each developer can then decide, based on their specific use case, whether to show cards on a subset of views.